qiskit/pulse/schedule.py

```diff
@@ -230,14 +231,30 @@ def filter(self, *filter_funcs: List[Callable],
                channels: Optional[Iterable[Channel]] = None,
                instruction_types: Optional[Iterable[Type['Instruction']]] = None,
                time_ranges: Optional[Iterable[Tuple[int, int]]] = None,
-               intervals: Optional[Iterable[Interval]] = None) -> 'Schedule':
+               intervals: Optional[Iterable[Interval]] = None,
+               filter_type: Optional[int] = 0) -> Union['Schedule', Tuple['Schedule', 'Schedule']]:
```

Comment:
Arbitrary integers are hard for the user to learn. The method name should tell the user what they can expect the method to do. It's also good for a function to do one thing reliably. Can you please make a new method that acts as a not filter? On the issue, I mentioned `exclude` as the method name, but other names could also work.